### ISSUE TITLE
Find out why model import got slower with recent sympy

### ISSUE THREAD

**Issue by dweindl** [ICB-DCM/AMICI#1107]
Partially recovered with #1104, but still not back at previous speed...

**Comment by dweindl** [comment]
Previously:
```
2020-05-13T13:52:51.7121762Z 2020-05-13 13:52:51.711 - amici.sbml_import - INFO - Finished processing SBML parameters          + (2.78E-01s)
2020-05-13T13:52:51.7136088Z 2020-05-13 13:52:51.713 - amici.sbml_import - INFO - Finished processing SBML compartments        + (1.18E-03s)
2020-05-13T13:52:56.9730701Z 2020-05-13 13:52:56.972 - amici.sbml_import - INFO - Finished processing SBML species             + (5.26E+00s)
2020-05-13T13:56:06.7128152Z 2020-05-13 13:56:06.712 - amici.sbml_import - INFO - Finished processing SBML reactions           + (1.90E+02s)
2020-05-13T13:56:25.1223430Z 2020-05-13 13:56:25.121 - amici.sbml_import - INFO - Finished processing SBML rules               + (1.84E+01s)
2020-05-13T13:56:25.2519939Z 2020-05-13 13:56:25.251 - amici.sbml_import - INFO - Finished processing SBML observables         + (2.43E-02s)
2020-05-13T13:59:04.7857067Z 2020-05-13 13:59:04.785 - amici.ode_export - INFO - Finished computing xdot                   +++++ (9.49E-03s)
2020-05-13T13:59:15.4729818Z 2020-05-13 13:59:15.472 - amici.ode_export - INFO - Finished computing dxdotdx                 ++++ (1.07E+01s)
2020-05-13T14:01:32.2250869Z 2020-05-13 14:01:31.843 - amici.ode_export - INFO - Finished computing w                     ++++++ (1.34E+02s)
2020-05-13T14:03:37.5443015Z 2020-05-13 14:03:37.543 - amici.ode_export - INFO - Finished computing dwdx                   +++++ (2.60E+02s)
2020-05-13T14:04:01.3522129Z 2020-05-13 14:04:01.351 - amici.ode_export - INFO - Finished computing dwdtcl                 +++++ (1.72E+01s)
2020-05-13T14:04:02.5496947Z 2020-05-13 14:04:02.549 - amici.ode_export - INFO - Finished computing dtcldx                 +++++ (1.20E+00s)
2020-05-13T14:04:03.5546060Z 2020-05-13 14:04:03.554 - amici.ode_export - INFO - Finished computing dwdx                    ++++ (2.86E+02s)
2020-05-13T14:04:59.9878242Z 2020-05-13 14:04:59.987 - amici.ode_export - INFO - Finished computing J                        +++ (3.55E+02s)
2020-05-13T14:05:09.5129697Z 2020-05-13 14:05:09.512 - amici.ode_export - INFO - Finished writing J.cpp                       ++ (3.65E+02s)
2020-05-13T14:05:31.7336892Z 2020-05-13 14:05:31.733 - amici.ode_export - INFO - Finished computing JB                       +++ (2.22E+01s)
2020-05-13T14:05:40.5188207Z 2020-05-13 14:05:40.518 - amici.ode_export - INFO - Finished writing JB.cpp                      ++ (3.10E+01s)
2020-05-13T14:05:40.5373068Z 2020-05-13 14:05:40.536 - amici.ode_export - INFO - Finished computing JDiag                    +++ (1.75E-02s)
2020-05-13T14:05:43.2011966Z 2020-05-13 14:05:43.200 - amici.ode_export - INFO - Finished writing JDiag.cpp                   ++ (2.68E+00s)
2020-05-13T14:05:43.2020006Z 2020-05-13 14:05:43.201 - amici.ode_export - INFO - Finished computing JSparse                  +++ (4.86E-05s)
2020-05-13T14:06:07.2836373Z 2020-05-13 14:06:07.283 - amici.ode_export - INFO - Finished writing JSparse.cpp                 ++ (2.41E+01s)
2020-05-13T14:06:07.2885331Z 2020-05-13 14:06:07.288 - amici.ode_export - INFO - Finished computing JSparseB                 +++ (4.78E-05s)
2020-05-13T14:06:30.7836699Z 2020-05-13 14:06:30.783 - amici.ode_export - INFO - Finished writing JSparseB.cpp                ++ (2.35E+01s)
2020-05-13T14:06:30.7892867Z 2020-05-13 14:06:30.788 - amici.ode_export - INFO - Finished computing Jy                       +++ (2.39E-04s)
2020-05-13T14:06:30.7919748Z 2020-05-13 14:06:30.791 - amici.ode_export - INFO - Finished writing Jy.cpp                      ++ (3.32E-03s)
2020-05-13T14:06:30.8080625Z 2020-05-13 14:06:30.807 - amici.ode_export - INFO - Finished computing dJydsigmay               +++ (1.53E-02s)
2020-05-13T14:06:30.8099009Z 2020-05-13 14:06:30.809 - amici.ode_export - INFO - Finished writing dJydsigmay.cpp              ++ (1.75E-02s)
2020-05-13T14:06:30.8147351Z 2020-05-13 14:06:30.814 - amici.ode_export - INFO - Finished computing dJydy                    +++ (3.99E-03s)
2020-05-13T14:06:30.8167958Z 2020-05-13 14:06:30.816 - amici.ode_export - INFO - Finished writing dJydy.cpp                   ++ (6.53E-03s)
2020-05-13T14:14:09.5012449Z 2020-05-13 14:14:09.500 - amici.ode_export - INFO - Finished computing dwdp                    ++++ (4.59E+02s)
2020-05-13T14:14:34.9894749Z 2020-05-13 14:14:34.989 - amici.ode_export - INFO - Finished computing dwdp                     +++ (4.84E+02s)
2020-05-13T14:17:04.2178506Z 2020-05-13 14:17:04.217 - amici.ode_export - INFO - Finished writing dwdp.cpp                    ++ (6.33E+02s)
2020-05-13T14:17:05.8200158Z 2020-05-13 14:17:05.819 - amici.ode_export - INFO - Finished writing dwdx.cpp                    ++ (1.60E+00s)
2020-05-13T14:17:38.4121513Z 2020-05-13 14:17:38.411 - amici.ode_export - INFO - Finished writing dxdotdw.cpp                 ++ (3.26E+01s)
2020-05-13T14:18:09.4109167Z 2020-05-13 14:18:09.410 - amici.ode_export - INFO - Finished computing dxdotdp_explicit         +++ (3.10E+01s)
2020-05-13T14:19:11.1160915Z 2020-05-13 14:19:11.115 - amici.ode_export - INFO - Finished writing dxdotdp_explicit.cpp        ++ (9.27E+01s)
2020-05-13T14:19:34.9985927Z 2020-05-13 14:19:34.998 - amici.ode_export - INFO - Finished computing dxdotdp_implicit          ++ (2.39E+01s)
2020-05-13T14:20:00.9126896Z 2020-05-13 14:20:00.912 - amici.ode_export - INFO - Finished computing y                      +++++ (2.51E-04s)
2020-05-13T14:20:01.0516873Z 2020-05-13 14:20:01.051 - amici.ode_export - INFO - Finished computing dydx                    ++++ (1.40E-01s)
2020-05-13T14:20:01.0984795Z 2020-05-13 14:20:01.098 - amici.ode_export - INFO - Finished computing dydw                    ++++ (3.58E-02s)
2020-05-13T14:20:01.1188049Z 2020-05-13 14:20:01.118 - amici.ode_export - INFO - Finished computing dydx                     +++ (2.08E-01s)
2020-05-13T14:20:01.1404014Z 2020-05-13 14:20:01.140 - amici.ode_export - INFO - Finished writing dydx.cpp                    ++ (2.30E-01s)
2020-05-13T14:20:01.5800478Z 2020-05-13 14:20:01.579 - amici.ode_export - INFO - Finished computing dydp                    ++++ (4.38E-01s)
2020-05-13T14:20:01.6018746Z 2020-05-13 14:20:01.601 - amici.ode_export - INFO - Finished computing dydp                     +++ (4.61E-01s)
2020-05-13T14:20:01.7470622Z 2020-05-13 14:20:01.746 - amici.ode_export - INFO - Finished writing dydp.cpp                    ++ (6.06E-01s)
2020-05-13T14:20:01.7483413Z 2020-05-13 14:20:01.748 - amici.ode_export - INFO - Finished computing sigmay                  ++++ (1.70E-04s)
2020-05-13T14:20:01.8895505Z 2020-05-13 14:20:01.889 - amici.ode_export - INFO - Finished computing dsigmaydp                +++ (1.42E-01s)
2020-05-13T14:20:02.0083781Z 2020-05-13 14:20:02.007 - amici.ode_export - INFO - Finished writing dsigmaydp.cpp               ++ (2.61E-01s)
2020-05-13T14:20:02.0091173Z 2020-05-13 14:20:02.008 - amici.ode_export - INFO - Finished writing sigmay.cpp                  ++ (4.73E-04s)
2020-05-13T14:20:03.1391655Z 2020-05-13 14:20:03.138 - amici.ode_export - INFO - Finished writing w.cpp                       ++ (1.13E+00s)
2020-05-13T14:20:03.1711857Z 2020-05-13 14:20:03.170 - amici.ode_export - INFO - Finished computing x0                       +++ (3.13E-02s)
2020-05-13T14:20:03.2703683Z 2020-05-13 14:20:03.269 - amici.ode_export - INFO - Finished writing x0.cpp                      ++ (1.31E-01s)
2020-05-13T14:20:04.7250055Z 2020-05-13 14:20:04.724 - amici.ode_export - INFO - Finished computing x0_fixedParameters       +++ (1.45E+00s)
2020-05-13T14:20:04.8136035Z 2020-05-13 14:20:04.813 - amici.ode_export - INFO - Finished writing x0_fixedParameters.cpp      ++ (1.54E+00s)
2020-05-13T14:23:10.4883972Z 2020-05-13 14:23:10.487 - amici.ode_export - INFO - Finished computing sx0                      +++ (1.86E+02s)
2020-05-13T14:23:25.5587443Z 2020-05-13 14:23:25.558 - amici.ode_export - INFO - Finished writing sx0.cpp                     ++ (2.01E+02s)
2020-05-13T14:28:02.8176281Z 2020-05-13 14:28:02.817 - amici.ode_export - INFO - Finished computing sx0_fixedParameters      +++ (2.77E+02s)
2020-05-13T14:28:56.9825092Z 2020-05-13 14:28:56.982 - amici.ode_export - INFO - Finished writing sx0_fixedParameters.cpp     ++ (3.31E+02s)
2020-05-13T14:28:59.1783854Z 2020-05-13 14:28:59.178 - amici.ode_export - INFO - Finished writing xdot.cpp                    ++ (2.20E+00s)
2020-05-13T14:28:59.1815852Z 2020-05-13 14:28:59.181 - amici.ode_export - INFO - Finished writing y.cpp                       ++ (2.82E-03s)
2020-05-13T14:28:59.1935698Z 2020-05-13 14:28:59.193 - amici.ode_export - INFO - Finished computing x_rdata                  +++ (1.13E-02s)
2020-05-13T14:28:59.4090207Z 2020-05-13 14:28:59.408 - amici.ode_export - INFO - Finished writing x_rdata.cpp                 ++ (2.27E-01s)
2020-05-13T14:28:59.4109105Z 2020-05-13 14:28:59.410 - amici.ode_export - INFO - Finished computing total_cl                 +++ (1.21E-03s)
2020-05-13T14:28:59.4384497Z 2020-05-13 14:28:59.438 - amici.ode_export - INFO - Finished writing total_cl.cpp                ++ (2.91E-02s)
2020-05-13T14:28:59.4474076Z 2020-05-13 14:28:59.447 - amici.ode_export - INFO - Finished computing x_solver                 +++ (8.24E-03s)
2020-05-13T14:28:59.6424686Z 2020-05-13 14:28:59.642 - amici.ode_export - INFO - Finished writing x_solver.cpp                ++ (2.03E-01s)
2020-05-13T14:29:12.4127280Z 2020-05-13 14:29:12.412 - amici.ode_export - INFO - Finished generating cpp code                  + (1.81E+03s)
2020-05-13T14:29:12.4697378Z 2020-05-13 14:29:12.469 - amici.petab_import - INFO - Finished Importing PEtab model                (2.18E+03s)
```

Now:
```
2020-05-29T16:22:57.0284272Z 2020-05-29 16:22:57.028 - amici.ode_export - INFO - Finished computing xdot                   +++++ (1.08E-02s)
2020-05-29T16:23:06.5113078Z 2020-05-29 16:23:06.510 - amici.ode_export - INFO - Finished computing dxdotdx                 ++++ (9.49E+00s)
2020-05-29T16:25:27.1295370Z 2020-05-29 16:25:27.129 - amici.ode_export - INFO - Finished computing w                     ++++++ (1.39E+02s)
2020-05-29T16:27:51.5145367Z 2020-05-29 16:27:51.514 - amici.ode_export - INFO - Finished computing dwdx                   +++++ (2.84E+02s)
2020-05-29T16:28:13.5926150Z 2020-05-29 16:28:13.592 - amici.ode_export - INFO - Finished computing dwdtcl                 +++++ (1.89E+01s)
2020-05-29T16:28:14.8646388Z 2020-05-29 16:28:14.864 - amici.ode_export - INFO - Finished computing dtcldx                 +++++ (1.27E+00s)
2020-05-29T16:28:36.2980325Z 2020-05-29 16:28:36.297 - amici.ode_export - INFO - Finished computing dwdx                    ++++ (3.28E+02s)
2020-05-29T16:29:25.7674239Z 2020-05-29 16:29:25.767 - amici.ode_export - INFO - Finished computing J                        +++ (3.89E+02s)
2020-05-29T16:29:35.3780366Z 2020-05-29 16:29:35.377 - amici.ode_export - INFO - Finished writing J.cpp                       ++ (3.98E+02s)
2020-05-29T16:29:55.5477730Z 2020-05-29 16:29:55.547 - amici.ode_export - INFO - Finished computing JB                       +++ (2.02E+01s)
2020-05-29T16:30:04.9361099Z 2020-05-29 16:30:04.935 - amici.ode_export - INFO - Finished writing JB.cpp                      ++ (2.96E+01s)
2020-05-29T16:30:04.9568185Z 2020-05-29 16:30:04.956 - amici.ode_export - INFO - Finished computing JDiag                    +++ (1.99E-02s)
2020-05-29T16:30:07.6567607Z 2020-05-29 16:30:07.656 - amici.ode_export - INFO - Finished writing JDiag.cpp                   ++ (2.72E+00s)
2020-05-29T16:30:07.6576278Z 2020-05-29 16:30:07.657 - amici.ode_export - INFO - Finished computing JSparse                  +++ (3.92E-05s)
2020-05-29T16:30:29.9111320Z 2020-05-29 16:30:29.910 - amici.ode_export - INFO - Finished writing JSparse.cpp                 ++ (2.23E+01s)
2020-05-29T16:30:29.9164703Z 2020-05-29 16:30:29.916 - amici.ode_export - INFO - Finished computing JSparseB                 +++ (3.84E-05s)
2020-05-29T16:30:51.7351967Z 2020-05-29 16:30:51.734 - amici.ode_export - INFO - Finished writing JSparseB.cpp                ++ (2.18E+01s)
2020-05-29T16:30:51.7405819Z 2020-05-29 16:30:51.740 - amici.ode_export - INFO - Finished computing Jy                       +++ (2.17E-04s)
2020-05-29T16:30:51.7432131Z 2020-05-29 16:30:51.743 - amici.ode_export - INFO - Finished writing Jy.cpp                      ++ (3.23E-03s)
2020-05-29T16:30:51.7586493Z 2020-05-29 16:30:51.758 - amici.ode_export - INFO - Finished computing dJydsigmay               +++ (1.46E-02s)
2020-05-29T16:30:51.7604015Z 2020-05-29 16:30:51.760 - amici.ode_export - INFO - Finished writing dJydsigmay.cpp              ++ (1.68E-02s)
2020-05-29T16:30:51.7652227Z 2020-05-29 16:30:51.765 - amici.ode_export - INFO - Finished computing dJydy                    +++ (3.98E-03s)
2020-05-29T16:30:51.7670878Z 2020-05-29 16:30:51.766 - amici.ode_export - INFO - Finished writing dJydy.cpp                   ++ (6.27E-03s)
2020-05-29T16:39:25.6944410Z 2020-05-29 16:39:25.694 - amici.ode_export - INFO - Finished computing dwdp                    ++++ (5.14E+02s)
2020-05-29T16:40:58.9228412Z 2020-05-29 16:40:58.922 - amici.ode_export - INFO - Finished computing dwdp                     +++ (6.07E+02s)
2020-05-29T16:43:08.6985619Z 2020-05-29 16:43:08.698 - amici.ode_export - INFO - Finished writing dwdp.cpp                    ++ (7.37E+02s)
2020-05-29T16:43:10.4628847Z 2020-05-29 16:43:10.462 - amici.ode_export - INFO - Finished writing dwdx.cpp                    ++ (1.76E+00s)
2020-05-29T16:43:46.4237966Z 2020-05-29 16:43:46.423 - amici.ode_export - INFO - Finished writing dxdotdw.cpp                 ++ (3.60E+01s)
2020-05-29T16:44:20.9206579Z 2020-05-29 16:44:20.920 - amici.ode_export - INFO - Finished computing dxdotdp_explicit         +++ (3.45E+01s)
2020-05-29T16:45:14.8134005Z 2020-05-29 16:45:14.813 - amici.ode_export - INFO - Finished writing dxdotdp_explicit.cpp        ++ (8.84E+01s)
2020-05-29T16:45:30.0474811Z 2020-05-29 16:45:30.047 - amici.ode_export - INFO - Finished computing dxdotdp_implicit          ++ (1.52E+01s)
2020-05-29T16:45:58.5608569Z 2020-05-29 16:45:58.560 - amici.ode_export - INFO - Finished computing y                      +++++ (1.91E-04s)
2020-05-29T16:45:58.7359689Z 2020-05-29 16:45:58.735 - amici.ode_export - INFO - Finished computing dydx                    ++++ (1.76E-01s)
2020-05-29T16:45:58.7620793Z 2020-05-29 16:45:58.761 - amici.ode_export - INFO - Finished computing dydw                    ++++ (2.41E-02s)
2020-05-29T16:45:58.7843501Z 2020-05-29 16:45:58.784 - amici.ode_export - INFO - Finished computing dydx                     +++ (2.25E-01s)
2020-05-29T16:45:58.8104284Z 2020-05-29 16:45:58.810 - amici.ode_export - INFO - Finished writing dydx.cpp                    ++ (2.52E-01s)
2020-05-29T16:45:59.3290557Z 2020-05-29 16:45:59.328 - amici.ode_export - INFO - Finished computing dydp                    ++++ (5.17E-01s)
2020-05-29T16:45:59.3608307Z 2020-05-29 16:45:59.360 - amici.ode_export - INFO - Finished computing dydp                     +++ (5.49E-01s)
2020-05-29T16:45:59.5252532Z 2020-05-29 16:45:59.524 - amici.ode_export - INFO - Finished writing dydp.cpp                    ++ (7.14E-01s)
2020-05-29T16:45:59.5265696Z 2020-05-29 16:45:59.526 - amici.ode_export - INFO - Finished computing sigmay                  ++++ (1.85E-04s)
2020-05-29T16:45:59.6737352Z 2020-05-29 16:45:59.673 - amici.ode_export - INFO - Finished computing dsigmaydp                +++ (1.48E-01s)
2020-05-29T16:45:59.8120902Z 2020-05-29 16:45:59.811 - amici.ode_export - INFO - Finished writing dsigmaydp.cpp               ++ (2.86E-01s)
2020-05-29T16:45:59.8129475Z 2020-05-29 16:45:59.812 - amici.ode_export - INFO - Finished writing sigmay.cpp                  ++ (4.74E-04s)
2020-05-29T16:46:01.1154168Z 2020-05-29 16:46:01.115 - amici.ode_export - INFO - Finished writing w.cpp                       ++ (1.30E+00s)
2020-05-29T16:46:01.1571321Z 2020-05-29 16:46:01.156 - amici.ode_export - INFO - Finished computing x0                       +++ (4.08E-02s)
2020-05-29T16:46:01.2718960Z 2020-05-29 16:46:01.271 - amici.ode_export - INFO - Finished writing x0.cpp                      ++ (1.56E-01s)
2020-05-29T16:46:02.9111817Z 2020-05-29 16:46:02.910 - amici.ode_export - INFO - Finished computing x0_fixedParameters       +++ (1.64E+00s)
2020-05-29T16:46:03.0271031Z 2020-05-29 16:46:03.026 - amici.ode_export - INFO - Finished writing x0_fixedParameters.cpp      ++ (1.75E+00s)
2020-05-29T16:49:40.8425528Z 2020-05-29 16:49:40.842 - amici.ode_export - INFO - Finished computing sx0                      +++ (2.18E+02s)
2020-05-29T16:49:59.1973538Z 2020-05-29 16:49:59.197 - amici.ode_export - INFO - Finished writing sx0.cpp                     ++ (2.36E+02s)
2020-05-29T17:05:17.4816257Z 2020-05-29 17:05:17.481 - amici.ode_export - INFO - Finished computing sx0_fixedParameters      +++ (9.18E+02s)
2020-05-29T17:06:25.2641000Z 2020-05-29 17:06:25.263 - amici.ode_export - INFO - Finished writing sx0_fixedParameters.cpp     ++ (9.86E+02s)
2020-05-29T17:06:28.4545527Z 2020-05-29 17:06:28.454 - amici.ode_export - INFO - Finished writing xdot.cpp                    ++ (3.19E+00s)
2020-05-29T17:06:28.4576884Z 2020-05-29 17:06:28.457 - amici.ode_export - INFO - Finished writing y.cpp                       ++ (2.62E-03s)
2020-05-29T17:06:28.4718933Z 2020-05-29 17:06:28.471 - amici.ode_export - INFO - Finished computing x_rdata                  +++ (1.34E-02s)
2020-05-29T17:06:28.7623111Z 2020-05-29 17:06:28.761 - amici.ode_export - INFO - Finished writing x_rdata.cpp                 ++ (3.04E-01s)
2020-05-29T17:06:28.7645954Z 2020-05-29 17:06:28.764 - amici.ode_export - INFO - Finished computing total_cl                 +++ (1.48E-03s)
2020-05-29T17:06:28.7997671Z 2020-05-29 17:06:28.799 - amici.ode_export - INFO - Finished writing total_cl.cpp                ++ (3.70E-02s)
2020-05-29T17:06:28.8241102Z 2020-05-29 17:06:28.823 - amici.ode_export - INFO - Finished computing x_solver                 +++ (2.35E-02s)
2020-05-29T17:06:29.0862974Z 2020-05-29 17:06:29.085 - amici.ode_export - INFO - Finished writing x_solver.cpp                ++ (2.86E-01s)
2020-05-29T17:06:44.4622190Z 2020-05-29 17:06:44.461 - amici.ode_export - INFO - Finished generating cpp code                  + (2.63E+03s)
2020-05-29T17:06:44.5202079Z 2020-05-29 17:06:44.519 - amici.petab_import - INFO - Finished Importing PEtab model                (3.07E+03s)
```


  |   |   | Previously | Now | Rel diff
-- | -- | -- | -- | -- | --
computing | xdot | +++++ | 0.00949 | 0.0108 | 0.14
computing | dxdotdx | ++++ | 10.7 | 9.49 | -0.11
computing | w | ++++++ | 134 | 139 | 0.04
computing | dwdx | +++++ | 260 | 284 | 0.09
computing | dwdtcl | +++++ | 17.2 | 18.9 | 0.10
computing | dtcldx | +++++ | 1.2 | 1.27 | 0.06
computing | dwdx | ++++ | 286 | 328 | 0.15
computing | J | +++ | 355 | 389 | 0.10
writing | J.cpp | ++ | 365 | 398 | 0.09
computing | JB | +++ | 22.2 | 20.2 | -0.09
writing | JB.cpp | ++ | 31 | 29.6 | -0.05
computing | JDiag | +++ | 0.0175 | 0.0199 | 0.14
writing | JDiag.cpp | ++ | 2.68 | 2.72 | 0.01
computing | JSparse | +++ | 4.86E-05 | 3.92E-05 | -0.19
writing | JSparse.cpp | ++ | 24.1 | 22.3 | -0.07
computing | JSparseB | +++ | 4.78E-05 | 3.84E-05 | -0.20
writing | JSparseB.cpp | ++ | 23.5 | 21.8 | -0.07
computing | Jy | +++ | 0.000239 | 0.000217 | -0.09
writing | Jy.cpp | ++ | 0.00332 | 0.00323 | -0.03
computing | dJydsigmay | +++ | 0.0153 | 0.0146 | -0.05
writing | dJydsigmay.cpp | ++ | 0.0175 | 0.0168 | -0.04
computing | dJydy | +++ | 0.00399 | 0.00398 | 0.00
writing | dJydy.cpp | ++ | 0.00653 | 0.00627 | -0.04
computing | dwdp | ++++ | 459 | 514 | 0.12
computing | dwdp | +++ | 484 | 607 | 0.25
writing | dwdp.cpp | ++ | 633 | 737 | 0.16
writing | dwdx.cpp | ++ | 1.6 | 1.76 | 0.10
writing | dxdotdw.cpp | ++ | 32.6 | 36 | 0.10
computing | dxdotdp_explicit | +++ | 31 | 34.5 | 0.11
writing | dxdotdp_explicit.cpp | ++ | 92.7 | 88.4 | -0.05
computing | dxdotdp_implicit | ++ | 23.9 | 15.2 | -0.36
computing | y | +++++ | 0.000251 | 0.000191 | -0.24
computing | dydx | ++++ | 0.14 | 0.176 | 0.26
computing | dydw | ++++ | 0.0358 | 0.0241 | -0.33
computing | dydx | +++ | 0.208 | 0.225 | 0.08
writing | dydx.cpp | ++ | 0.23 | 0.252 | 0.10
computing | dydp | ++++ | 0.438 | 0.517 | 0.18
computing | dydp | +++ | 0.461 | 0.549 | 0.19
writing | dydp.cpp | ++ | 0.606 | 0.714 | 0.18
computing | sigmay | ++++ | 0.00017 | 0.000185 | 0.09
computing | dsigmaydp | +++ | 0.142 | 0.148 | 0.04
writing | dsigmaydp.cpp | ++ | 0.261 | 0.286 | 0.10
writing | sigmay.cpp | ++ | 0.000473 | 0.000474 | 0.00
writing | w.cpp | ++ | 1.13 | 1.3 | 0.15
computing | x0 | +++ | 0.0313 | 0.0408 | **0.30**
writing | x0.cpp | ++ | 0.131 | 0.156 | 0.19
computing | x0_fixedParameters | +++ | 1.45 | 1.64 | 0.13
writing | x0_fixedParameters.cpp | ++ | 1.54 | 1.75 | 0.14
computing | sx0 | +++ | 186 | 218 | 0.17
writing | sx0.cpp | ++ | 201 | 236 | 0.17
computing | sx0_fixedParameters | +++ | 277 | 918 | **2.31**
writing | sx0_fixedParameters.cpp | ++ | 331 | 986 | **1.98**
writing | xdot.cpp | ++ | 2.2 | 3.19 | **0.45**
writing | y.cpp | ++ | 0.00282 | 0.00262 | -0.07
computing | x_rdata | +++ | 0.0113 | 0.0134 | 0.19
writing | x_rdata.cpp | ++ | 0.227 | 0.304 | **0.34**
computing | total_cl | +++ | 0.00121 | 0.00148 | 0.22
writing | total_cl.cpp | ++ | 0.0291 | 0.037 | 0.27
computing | x_solver | +++ | 0.00824 | 0.0235 | **1.85**
writing | x_solver.cpp | ++ | 0.203 | 0.286 | **0.41**
generating | cpp code | + | 1810 | 2630 | **0.45
Importing | PEtab model |   | 2180 | 3070 | **0.41**

Seems like `sx0_fixedParameters` still needs some attention...

**Comment by dweindl** [comment]
As also "writing sx0_fixedParameters.cpp" got significantly longer, I guess there are still some 0-derivatives produced somewhere...

**Comment by dweindl** [comment]
> As also "writing sx0_fixedParameters.cpp" got significantly longer, I guess there are still some 0-derivatives produced somewhere...

Yes, file contains 7e5 lines of ... = 0.

**Comment by FFroehlich** [comment]
> > As also "writing sx0_fixedParameters.cpp" got significantly longer, I guess there are still some 0-derivatives produced somewhere...
> 
> Yes, file contains 7e5 lines of ... = 0.

🙈

**Comment by dweindl** [comment]
Earlier: 
2020-05-13T14:29:12.4697378Z 2020-05-13 14:29:12.469 - amici.petab_import - INFO - Finished Importing PEtab model                (2.18E+03s)

With #1107:
2020-05-29 22:10:42.790 - amici.petab_import - INFO - Finished Importing PEtab model                (2.01E+03s)

Code generation 3min faster than before. Considering closed with #1107.
